apiservers/engine/backends/container.go

```diff
@@ -20,90 +20,111 @@ import (
 	"io"
 	"net/http"
 	"os"
-	goexec "os/exec"
-	"sync"
+	"os/exec"
 	"time"
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/docker/docker/api/types/backend"
 	derr "github.com/docker/docker/errors"
 	"github.com/docker/docker/pkg/archive"
+	"github.com/docker/docker/pkg/stdcopy"
 	"github.com/docker/docker/pkg/version"
 	"github.com/docker/engine-api/types"
 	"github.com/docker/engine-api/types/container"
 	"github.com/docker/engine-api/types/strslice"
 
+	viccontainer "github.com/vmware/vic/apiservers/engine/backends/container"
 	"github.com/vmware/vic/apiservers/portlayer/client/containers"
+	"github.com/vmware/vic/apiservers/portlayer/client/interaction"
 	"github.com/vmware/vic/apiservers/portlayer/client/scopes"
 	"github.com/vmware/vic/apiservers/portlayer/client/storage"
 	"github.com/vmware/vic/apiservers/portlayer/models"
+	"github.com/vmware/vic/pkg/attachutils"
 	"github.com/vmware/vic/pkg/trace"
 )
 
-type V1Compatibility struct {
-	ID     string
-	Config container.Config
-}
-
 type Container struct {
 	ProductName string
-
-	// Protects following map
-	m sync.Mutex
-
-	// FIXME: in-memory map to keep image name to vmdk name relationship
-	HackMap map[string]V1Compatibility
 }
 
 // docker's container.execBackend
 
+// ContainerExecCreate sets up an exec in a running container.
 func (c *Container) ContainerExecCreate(config *types.ExecConfig) (string, error) {
 	return "", fmt.Errorf("%s does not implement container.ContainerExecCreate", c.ProductName)
 }
 
+// ContainerExecInspect returns low-level information about the exec
+// command. An error is returned if the exec cannot be found.
 func (c *Container) ContainerExecInspect(id string) (*backend.ExecInspect, error) {
 	return nil, fmt.Errorf("%s does not implement container.ContainerExecInspect", c.ProductName)
 }
 
+// ContainerExecResize changes the size of the TTY of the process
+// running in the exec with the given name to the given height and
+// width.
 func (c *Container) ContainerExecResize(name string, height, width int) error {
-	return fmt.Errorf("%s does not implement container.ContainerExecResize", c.ProductName)
+	return nil
```

Comment:
If we don't support it yet, then should we revert this line and return the error or is that error getting in the way somewhere else?